cgmanifest.json

```diff
@@ -21609,6 +21609,16 @@
         }
       }
     },
+    {
+      "component": {
+        "type": "other",
+        "other": {
+          "name": "python-retrying",
+          "version": "1.3.3",
+          "downloadUrl": "%{url}/archive/refs/tags/v1.3.3.tar.gz"
```

Comment:
Please change to not refer to RPM macros

Reply:
Done. My bad I introduced this silly typo.